packages/flutter/test/material/text_field_test.dart

```diff
@@ -210,6 +210,43 @@ void main() {
     );
   }
 
+  testWidgets(
+    'Live Text button shows and hide correctly when LiveTextStatus changes.',
+    (WidgetTester tester) async {
+      final TextEditingController controller = TextEditingController(text: '');
+      const Key key = ValueKey<String>('TextField');
+      final FocusNode focusNode = FocusNode();
+      final Widget app = MaterialApp(
+        theme: ThemeData(platform: TargetPlatform.iOS),
+        home: Scaffold(
+          body: Center(
+            child: TextField(
+              key: key,
+              controller: controller,
+              focusNode: focusNode,
+            ),
+          ),
+        ),
+      );
+
+      mockClipboard.mockLiveTextInputEnabled = true;
```

Comment:
This probably doesn't belong on mockClipboard.  I think you can use the same pattern of handleMethodCall though.  The overall approach in this test is great 👍